ui/src/components/ESSelector/HitsSearch.js

```diff
@@ -47,12 +53,17 @@ export class HitsSearch extends Component {
         ? searchQuery + '*'
         : searchQuery;
       const response = await this.props.query(queryString);
-      const results = [];
-      for (let hit of response.data.hits) {
-        results.push(serialize(hit));
-      }
-      if (this.props.onResults) {
-        this.props.onResults(results);
+      let results = [];
```

Comment:
Should you call `this.props.onResults` even if `serialize` is `false` or not?

```
results = serialize ? response.data.hits.map(hit => serialize(hit)) : response.data.hits;
if (this.props.onResults) {
  this.props.onResults(results);
}
```

Reply:
actually I have checked and it was a leftover from something els, I removed it.